tools/sambamba/sambamba.xml

```diff
@@ -0,0 +1,226 @@
+<tool id="sambamba" name="sambamba" version="0.5.9.0">
+    <description>Sambamba: Process your BAM data faster!</description>
+    <requirements>
+        <requirement type="package" version="1.0">sambamba</requirement>
```

Comment:
This refers to a `tool_dependencies.xml` which I believe is missing. Could you add it? It should look something like this: https://bitbucket.org/lance_parsons/sambamba_filter_galaxy_wrapper/src/fa4adadc5e8bc444b2278dc4f8ad8dfd1fd1f55f/tool_dependencies.xml?fileviewer=file-view-default

Also I think `1.0` should be `0.5.9` as you did in the first line of this file